.travis.yml

```diff
@@ -65,9 +65,9 @@ jobs:
   - name: 'End-to-end Tests: Chrome beta'
     env: CV_BUILD=test CV_BROWSER=Chrome_travis CV_VERSION=beta
     script: *2
-  - name: 'End-to-end Tests: Chrome unstable'
-    env: CV_BUILD=test CV_BROWSER=Chrome_travis CV_VERSION=unstable
-    script: *2
+#  - name: 'End-to-end Tests: Chrome unstable'
+#    env: CV_BUILD=test CV_BROWSER=Chrome_travis CV_VERSION=unstable
+#    script: *2
```

Comment:
Does it still has to be removed - or should it stay?

Reply:
I don't know, lets give it a try.

Reply:
It seems that this is still necessary.